packages/friends/yml/en/robert-barclay.yml

```diff
@@ -2,6 +2,26 @@ id: 105b22b5-0d97-422d-821f-becefa9482a0
 name: Robert Barclay
 slug: robert-barclay
 gender: male
+born: 1648
+died: 1690
+quotes:
+  - source: William Sewel
+    text:
+      Though [Robert Barclay's] natural abilities were great enough to have made him
+      surpass others in human learning, and so to have become famous among men, yet he so
+      little valued that knowledge, that he in no way endeavored to be distinguished on
+      that account. But his chief aim was to advance in real godliness, as the fellowship
+      I had with him has undoubtedly assured me; for I was well acquainted with him.
+  - source: Robert Barclay
+    text:
+      For what I have written comes more from my heart than from my head; from what I have
+      heard with the ears of my soul and seen with my inward eyes, and my hands have
+      handled of the Word of Life. What has been inwardly manifested to me of the things
+      of God, that do I declare; not so much minding the eloquence and excellency of
+      speech, as desiring to demonstrate the efficacy and operation of Truth.
+residences:
+  - city: York
+    region: England
```

Comment:
Is this just a copy paste error? Should be Ury Scotland, or something like that, right?